fix for: error: 'numeric_limits' is not a member of 'std'
author: Kenneth Hoste (HPC-UGent)
--- popscle-da70fc78da385ef049e0e890342acfd62842cae0/gtf_interval_tree.h.orig	2021-05-05 06:49:20.000000000 +0200
+++ popscle-da70fc78da385ef049e0e890342acfd62842cae0/gtf_interval_tree.h	2023-07-05 09:13:54.273004779 +0200
@@ -29,6 +29,7 @@
 #include <iostream>
 #include <memory>
 #include <cassert>
+#include <limits>
 
 template <class Scalar, typename Value>
 class gtfInterval {
